### ISSUE TITLE
firmware question,

### ISSUE THREAD

**Issue by FrostKnight** [NordicPlayground/nrf70-linux-driver#43]
I see most of this is under free licenses, I just am curious:

Where the firmware is and what is the license of the firmware?

I just was curious if this is fully free or as some would say, fully open source. I differ on which I prefer to say, but yeah, let me know.

**Comment by scottcopus** [comment]
https://github.com/NordicPlayground/nrf70-linux-driver?tab=readme-ov-file#known-issues
I don't know much about Nordic as most of my experience is with Espressif, but does anyone know if Nordic makes continuous bug fixes or improvements to their SOC firmwares like the nrf70*?  From a semi-noob's perspective, when someone flashes a program to one of the Nordic SOCs, does that ever resolve any of those low-level wifi or other proto issues... in case any of those "known issues" (linked here) is caused by it?

**Comment by maxd-nordic** [comment]
@scottcopus Yes, Nordic updates their whole software offering continuously. This includes binary blobs like softdevices and the wifi firmware. The known issues are what is seen with the currently released version and will likely be resolved in the next release.

**Comment by FrostKnight** [comment]
> @scottcopus Yes, Nordic updates their whole software offering continuously. This includes binary blobs like softdevices and the wifi firmware. The known issues are what is seen with the currently released version and will likely be resolved in the next release.

Here is another question that occurred to me, does that binary blob you mentioned, require anything on the software end to make it work? And does it do any remote dialing, like surveillance nonsense. Just wondered.

**Comment by scottcopus** [comment]
> @scottcopus Yes, Nordic updates their whole software offering continuously. This includes binary blobs like softdevices and the wifi firmware. The known issues are what is seen with the currently released version and will likely be resolved in the next release.

@maxd-nordic Thanks for the info. I'm familiar with Espressif devices and ESP-IDF/Arduino/PlatformIO IDEs. For Nordic chips, when you upload the firmware/program/"sketch", are those binary blobs (wifi library/etc) part of that same upload? Or are those blobs they a separate part of the firmware? If they're separate, how are those updated? Thanks

**Comment by maxd-nordic** [comment]
@scottcopus They are compiled into the app. Not entirely sure how this is done in the linux driver (I guess it bundles the fw as well), but on NCS/Zephyr, you'll get the latest version automatically when updating to a newer release.
